schema/openapi.yaml

```diff
@@ -65,6 +78,12 @@ paths:
       parameters:
         - $ref: '#/components/parameters/offsetParam'
         - $ref: '#/components/parameters/limitParam'
+        - $ref: '#/components/parameters/ref_filter'
+        - $ref: '#/components/parameters/repo_filter'
+        - $ref: '#/components/parameters/cluster_filter'
+        - $ref: '#/components/parameters/image_filter'
+        - $ref: '#/components/parameters/start_date'
+        - $ref: '#/components/parameters/end_date'
```

Comment:
May be nit-picky here, but you could change the names here to camel-case to match the others.